integ-test/src/test/java/org/opensearch/sql/ppl/DateTimeFunctionIT.java

```diff
@@ -6,8 +6,11 @@
 
 package org.opensearch.sql.ppl;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Are these imports used?

Reply:
Assert equals is in line 852. (there was another import that wasn't used, now removed, thanks).